Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -555,6 +556,104 @@ def get_file_data(self, file_hash):
         cmd_url = f'/files/{file_hash}'
         return self.ms_client.http_request(method='GET', url_suffix=cmd_url)
 
+    def list_indicators(self, indicator_id: Optional[str] = None) -> List:
+        """Lists indicators. if indicator_id supplied, will get only that indicator.
+
+        Args:
+            indicator_id: if provided, will get only this specific id.
+
+        Returns:
+            json of a response.
+        """
+        cmd_url = self.indicators_endpoint
+        # For getting one indicator
+        if indicator_id is not None:
+            cmd_url = urljoin(cmd_url, indicator_id)
+        # TODO: check in the future if the filter is working. Then remove the filter function.
+        # params = {'$filter': 'targetProduct=\'Microsoft Defender ATP\''}
+        resp = self.ms_client.http_request(
+            'GET', full_url=cmd_url, url_suffix=None, scope='graph', timeout=1000,
+            ok_codes=(200, 204, 206, 404), resp_type='response'
+        )
+        # 404 - No indicators found, an empty list.
+        if resp.status_code == 404:
+            return []
+        resp = resp.json()
+        # If 'value' is in the response, should filter and limit. The '@odata.context' key is in the root which we're
+        # not returning
+        if 'value' in resp:
+            resp['value'] = list(
+                filter(lambda item: item.get('targetProduct') == 'Microsoft Defender ATP', resp.get('value', []))
+            )
+            resp = resp['value']
+        # If a single object - should remove the '@odata.context' key.
+        else:
+            resp.pop('@odata.context')
+            resp = [resp]
+        return [assign_params(values_to_ignore=[None], **item) for item in resp]
+
+    def create_indicator(self, body: Dict) -> Dict:
+        """Creates indicator from the given body.
+
+        Args:
+            body: Body represents an indicator.
+
+        Returns:
+            A response from the API.
+        """
+        resp = self.ms_client.http_request(
+            'POST', full_url=self.indicators_endpoint, json_data=body, url_suffix=None, scope='graph'
+        )
+        # A single object - should remove the '@odata.context' key.
+        resp.pop('@odata.context')
```

Comment:
why are we doing this ?

Reply:
Because it's a constant value with no use. Can consult @Arsenikr if we need to keep it.

Reply:
got a verdict if there's value returning this in the raw response ? @Arsenikr  ?

Reply:
@jochman @Arsenikr what about this one?

Reply:
you can remove this